### PR TITLE
Allow runc on the skydive deployment.

### PR DESCRIPTION
**Pull request by SchSeba**
**What this PR does / why we need it**:
Support cri-o engine for the skydive deployment

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Support cri-o engine in skydive.
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt-ansible/529)

### CODE REVIEW

**Comment by SchSeba** [comment]
@gbenhaim Can you please take a look I think we can enable the skydive again I also change the elasticsearch max ram to 256M so it will not kill the ci environment

File: automation/check-patch.sh [hunk 1]
```diff
@@ -205,12 +205,14 @@ run() {
...
         "provider=$provider"
         "inventory_file=$inventory_file"
+        "cluster=$cluster"
         "ansible_modules_version=$ansible_modules_version"
         "kubevirt_openshift_version=$kubevirt_openshift_version"
         "openshift_playbook_path=$openshift_playbook_path"
         "storage_role=$storage_role"
         "cluster=$cluster"
         "platform=$cluster"
+        "deploy_skydive=true"
```
Comment (myakove) [inline automation/check-patch.sh]: Please move this to a different  PR so we can merge this one without @gbenhaim

Comment (SchSeba) [reply automation/check-patch.sh]: Done lets wait for the CI

**Comment by SchSeba** [comment]
@myakove please take a look I think you can merge it now.

**Pull request merged**